pkgs/tools/misc/coreutils/default.nix

```diff
@@ -16,7 +16,7 @@
 , minimal ? true
 , withOpenssl ? !minimal, openssl
 , withPrefix ? false
-, singleBinary ? "symlinks" # you can also pass "shebangs" or false
+, singleBinary ? if stdenv.isFreeBSD then false else "symlinks" # you can also pass "shebangs" or false
```

Comment:
Does shebangs work?

Reply:
looks like it does, thanks for the idea!

Reply:
Looks like it does not work. FreeBSD doesn't seem to support using interpreted scripts as interpreters, so anything with coreutils/bin/env as the interpreter will fail. I'm debugging symlinks right now to see what the deal is.

Reply:
Best I can tell, coreutils is busted for the single-binary build, and freebsd just happens to trigger it. [This](https://github.com/coreutils/coreutils/blob/master/src/local.mk#L485) seems to be the root cause - mixing DEPENDENCIES (i.e. make target prerequisites) and LDADD (i.e. flags for ld) is a really bad idea, since suddenly you have -lintl as a make target.

I don't know why shebangs was working two days ago. Maybe suddenly stripping all our libs causes coreutils configure to detect different LDADD flags?

I'm going to revert to singleBinary = false, submit an upstream issue, link to said upstream issue, and move on with my life.